Documentation/gettingstarted/cassandra.rst

```diff
@@ -0,0 +1,329 @@
+
+  (11 rows)
+
+
+Uh oh!  The rebels now has strategic information about empire troop strengths at each location in the galaxy.  
+
+But even more nasty from a security perspective is that the outpost container can also access information in any keyspace, 
+including the deathstar keyspace.  For example, run:
+
+::
+
+ $ cqlsh> SELECT * FROM deathstar.scrum_notes;
+
+  empire_member_id                     | content                                                                                                        | creation
+ --------------------------------------+----------------------------------------------------------------------------------------------------------------+--------------------------------------
+ 34e564c2-781b-477e-acd0-b357d67f94f2 | Designed protective shield for deathstar.  Could be based on nearby moon.  Feature punted to v2.  Not blocked. | c3c8b210-bf57-11e8-98e6-f1a9f45fc4d8
+ dfa974ea-88cd-4e9b-85e3-542b9d00e2df |   I think the exhaust port could be vulnerable to a direct hit.  Hope no one finds out about it.  Not blocked. | c37f4d00-bf57-11e8-98e6-f1a9f45fc4d8
+ ee12306a-7b44-46a4-ad68-42e86f0f111e |        Trying to figure out if we should paint it medium grey, light grey, or medium-light grey.  Not blocked. | c32daa90-bf57-11e8-98e6-f1a9f45fc4d8
+
+ (3 rows)
+
+We see that any outpost can actually access the deathstar scrum notes, which mentions a pretty serious issue with the exhaust port.   
+
+Step 5: Securing Access to Cassandra with Cilium
+================================================
+
+Obviously, it would be much more secure to limit each pod's access to the Cassandra server to be
+least privilege (i.e., only what is needed for the app to operate correctly and nothing more).
+
+We can do that with the following Cilium security policy.   As with Cilium HTTP policies, we can write
+policies that identify pods by labels, and then limit the traffic in/out of this pod.  In
+this case, we'll create a policy that identifies the tables that each client should be able to access, 
+the actions that are allowed on those tables, and deny the rest.
+
+As an example, a policy could limit containers with label *app=empire-outpost* to only be able to 
+insert entries into the table "attendance.daily_reports", but would block any attempt by a compromised outpost
+to read all attendance information or access other keyspaces. 
+
+.. image:: images/cilium_cass_gsg_attack.png
+
+Here is the *CiliumNetworkPolicy* rule that limits access of pods with label *app=empire-outpost* to
+only consume on topic *empire-announce*:
+
+.. literalinclude:: ../../examples/kubernetes-cassandra/cass-sw-security-policy.yaml
+
+A *CiliumNetworkPolicy* contains a list of rules that define allowed requests, meaning that requests
+that do not match any rules are denied as invalid.
+
+The rule explicitly matches Cassandra connections destined to TCP 9042 on cass-server pods, and allows consume/produce actions on various topics of interest.
```

Comment:
Non-blocking: Maybe instead of "consume / produce" and "topics", refer to the Cassandra terminology (select, update, etc.)?